### PR TITLE
Feat/image input

### PR DESCRIPTION
**Pull request by Aditi0505**
# Description

- Added Image and Input Component
- Added Image and Input Documentation

# Screenshots

## Responsive and rounded images
image
## Input for type - text, email, password, address, and mobile.
image
## Disabled Input
image
## validation styles
image

### CODE REVIEW

**Review by Naveenchandar — APPROVED** [review]
Looks good

File: docs/docs-style.css [hunk 1]
```diff
@@ -4,6 +4,8 @@
 @import url("../components/alert/alert.css");
 @import url("../components/button/button.css");
 @import url("../components/card/card.css");
+@import url("../components/image/image.css");
+@import url("../components/input/input.css");
```
Comment (aniketsingh13) [inline docs/docs-style.css:8]: you can put them in  one CSS file

**Review by aniketsingh13 — APPROVED** [review]
good to go you can consider the changes

File: docs/docs-style.css (hunk 1 above)
Comment (Aditi0505) [reply docs/docs-style.css:8]: Sure, thanks for suggesting this. I will make this change at the end.

**Pull request merged**